charts/orb/Chart.yaml

```diff
@@ -25,50 +25,50 @@ dependencies:
     repository: "@jaegertracing"
     condition: jaeger-operator.enabled
   - name: nats
-    version: "6.4.2"
+    version: "7.0.4"
```

Comment:
still major version, they don't have anything in 6.x? if not, and we must go to 7.x, then we have to test first.